src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -54,7 +54,6 @@ public partial class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.Reso
             Mandatory = false,
             Position = 2,
             ValueFromPipelineByPropertyName = true)]
-        [CmdletParameterBreakingChange(nameof(DiskSizeGB), ChangeDescription = "'DiskSizeInBytes' instead of 'DiskSizeGB' is going to be used when CreateOption is 'Upload'.")]
         public int DiskSizeGB { get; set; }
```

Comment:
I don't find `DiskSizeInBytes`. Should you rename `DiskSizeGB` to `DiskSizeInBytes`?

Reply:
Actually the description was wrong.  It was supposed to be 'UploadSizeInBytes'.  'UploadSizeInBytes' is already one of the parameters of New-AzDiskConfig.
Breaking change is, when CreateOption = Upload, a user should specify 'UploadSizeInBytes', instead of 'DiskSizeGB'